bitnami/rabbitmq/templates/statefulset.yaml

```diff
@@ -290,69 +224,46 @@ spec:
               - key: tls.key
                 path: server_key.pem
         {{- end }}
-        {{- if .Values.rabbitmq.communityPlugins }}
-        - name: {{ template "rabbitmq.fullname" . }}-plugins
-          configMap:
-            name: {{ template "rabbitmq.fullname" . }}-plugins
-        {{- end }}
-        - name: config-volume
+        - name: configuration
           configMap:
             name: {{ template "rabbitmq.fullname" . }}-config
             items:
               - key: rabbitmq.conf
                 path: rabbitmq.conf
-              {{- if .Values.rabbitmq.advancedConfiguration}}
-              - key: advanced.config
-                path: advanced.config
+              {{- if .Values.advancedConfiguration}}
+              - key: advanced.conf
+                path: advanced.conf
               {{- end }}
-              - key: enabled_plugins
-                path: enabled_plugins
-        - name: healthchecks
-          configMap:
-            name: {{ template "rabbitmq.fullname" . }}-healthchecks
-            items:
-              - key: rabbitmq-health-check
-                path: rabbitmq-health-check
-                mode: 111
-              - key: rabbitmq-api-check
-                path: rabbitmq-api-check
-                mode: 111
-        {{- if .Values.rabbitmq.loadDefinition.enabled }}
+        {{- if .Values.loadDefinition.enabled }}
         - name: load-definition-volume
           secret:
-            secretName: {{ tpl .Values.rabbitmq.loadDefinition.secretName . | quote }}
+            secretName: {{ tpl .Values.loadDefinition.existingSecret . | quote }}
         {{- end }}
         {{- if .Values.extraVolumes }}
         {{- toYaml .Values.extraVolumes | nindent 8 }}
         {{- end }}
-      {{- if not .Values.persistence.enabled }}
+  {{- if not .Values.persistence.enabled }}
         - name: data
           emptyDir: {}
-      {{- else if .Values.persistence.existingClaim }}
+  {{- else if .Values.persistence.existingClaim }}
         - name: data
           persistentVolumeClaim:
-          {{- with .Values.persistence.existingClaim }}
+            {{- with .Values.persistence.existingClaim }}
             claimName: {{ tpl . $ }}
-          {{- end }}
+            {{- end }}
   {{- else }}
   volumeClaimTemplates:
     - metadata:
         name: data
-        labels:
-          app: {{ template "rabbitmq.name" . }}
-          release: "{{ .Release.Name }}"
-          heritage: "{{ .Release.Service }}"
+        labels: {{- include "common.labels.standard" . | nindent 10 }}
```

Comment:
### IMPORTANT

This helper contains the chart version, so it will change when upgrading and therefore mutating the volumeClaimTemplate field. This will cause all upgrades to fail

Reply:
Good catch